pkgs/misc/vim-plugins/generated.nix

```diff
@@ -7013,6 +7025,18 @@ final: prev:
     meta.homepage = "https://github.com/vim-autoformat/vim-autoformat/";
   };
 
+  vim-autoformat = buildVimPluginFrom2Nix {
+    pname = "vim-autoformat";
+    version = "2021-11-07";
+    src = fetchFromGitHub {
+      owner = "vim-autoformat";
+      repo = "vim-autoformat";
+      rev = "bb11f30377985e45e2eecef570856d42dbabb8b0";
+      sha256 = "1lx5lrb0hjijvwngvgsgchww70c1bgqf5qxj8lhvk11dvxn3k087";
+    };
+    meta.homepage = "https://github.com/vim-autoformat/vim-autoformat/";
+  };
+
```

Comment:
this needs to be removed, as it's already defined

Reply:
My bad. Forgot that update.py does not filter-out duplicates. Fixed